### PR TITLE
QAE-301 - E2E test case for blog archive meta options

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for blog archive meta options

### Screenshots

https://share.getcloudapp.com/Qwu4RXyq

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/blog/archive/meta.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/blog/archive/meta.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'blog archive in the customizer', () => {
+	it( 'post structure structure should apply corectly', async () => {
+		const BlogpostStructure = {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/archive/meta.test.js:5]:
```suggestion
		const BlogPostStructure = {
```

File: tests/e2e/specs/customizer/blog/archive/meta.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'blog archive in the customizer', () => {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/archive/meta.test.js:3]: first letter should be capital

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/archive/meta.test.js]: Done

File: tests/e2e/specs/customizer/blog/archive/meta.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/archive/meta.test.js]: Done

File: tests/e2e/specs/customizer/blog/archive/meta.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
[the remark this answers is not in the document]
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/archive/meta.test.js]: Done

File: tests/e2e/specs/customizer/blog/archive/meta.test.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+				3: 'date',
+			},
+		};
+		await setCustomize( blogPostStructure );
+		let ppStatus = false;
+		while ( false === ppStatus ) {
+			await createNewPost( { postType: 'post', title: 'test' } );
+			ppStatus = await publishPost();
+		}
+		await page.goto( createURL( '/author/admin' ), {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/blog/archive/meta.test.js:20]: Showing selector-error:-https://share.bsf.io/NQuN4J1x

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/archive/meta.test.js]: Done

**Review by bsfsnehal — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**